src/preferences/colorpaletteeditormodel.h

```diff
@@ -46,3 +49,25 @@ class ColorPaletteEditorModel : public QStandardItemModel {
     bool m_bEmpty;
     bool m_bDirty;
 };
+
+class HotcueIndexListItem : public QStandardItem {
+    Q_OBJECT
```

Comment:
Remove this macro and  it will build.

Reply:
It currently doesn't build because of a bunch of syntax and typeerrors. I don't see how removing it would suddenly fix the build. Can I still remove the `Q_OBJECT` though? It doesn't use any signals/slots or moc specific mechisms, though the parent QStandardItem does.

Reply:
Q_OBJECT macro defines some things in objects Inherited from QObject. QStandardItem does not inherit from QObject.